docs/hardware.rst

```diff
@@ -82,10 +82,10 @@ Workstations
   <https://tails.boum.org/news/Tails_3.0_will_require_a_64-bit_processor/index.en.html>`_.
 
 These components are necessary to do the initial installation of
-SecureDrop and to process submissions using the airgapped workflow.
+SecureDrop and to process submissions using the air-gapped workflow.
```

Comment:
I agree we should use it; having said that, this would need to be changed in other docs, as well, so I can scope a separate PR to make docs-wide changes like this, if that makes the most sense.